src/main/java/com/timelogr/enterprise/TimeLogrController.java

```diff
@@ -1,55 +1,57 @@
 package com.timelogr.enterprise;
 
+import com.timelogr.enterprise.service.ITimeLogrService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-//ALL WEB FILES GO IN THE resources/templates DIRECTORY
+// ALL WEB FILES GO IN THE resources/templates DIRECTORY
 @Controller
 @RequestMapping("/")
 public class TimeLogrController {
-    private TimeLogrServices timeLogrServices;
 
+    private final ITimeLogrService timeLogrService;
 
     @Autowired
-    public TimeLogrController(TimeLogrServices timeLogrServices){
+    public TimeLogrController(ITimeLogrService timeLogrService){
         super();
-        this.timeLogrServices = timeLogrServices;
+        this.timeLogrService = timeLogrService;
     }
```

Comment:
Changed the code to use the ITimeLogr interface instead of the service (directly)

Reply:
Good idea.  Better for polymorphism.